miniapps/dpg/acoustics.cpp

```diff
@@ -117,9 +117,11 @@ int main(int argc, char *argv[])
    real_t rnum=1.0;
    int ref = 0;
    bool static_cond = false;
+   int visport = 19916;
    int iprob = 0;
 
    OptionsParser args(argc, argv);
+   args.AddOption(&visport, "-p", "--send-port", "Socket for GLVis.");
```

Comment:
This should ideally be in the same order as the variables above, so currently the last `AddOption`